client/js/components/map/publicdetail/Map.vue

```diff
@@ -607,25 +601,18 @@ export default {
       })
 
       if (serviceType === 'wmts') {
-        // Delete custom Ajax headers as they may not be allowed by cors
-        this.deleteCustomAjaxHeaders()
         const layerArray = Array.isArray(layer.attributes.layers) ? layer.attributes.layers : layer.attributes.layers.split(',')
         const url = this.addGetCapabilityParamToUrl(layer.attributes.url)
-        $.ajax({
-          dataType: 'xml',
-          url: url || '',
-          async: false,
-          success: response => {
+        externalApi(url)
+          .then((response) => {
```

Comment:
```suggestion
          .then(response => {
```

Code looks legit, but to be safe: Did You test that?

Reply:
I did, but only a hacky version with WMS layers instead of WMTS, because I couldn't find a WMTS source, that works. Do you know one?

Reply:
Ok, I found one, something does not work right. I'm on it.

Reply:
Please check first if it worked before. Maybe it doesn't have anything todo with the refactoring. Then we should create a bug Ticket but not try to fix it within this PR

Reply:
It worked before, but now it does not. We get a different response, but I don't understand why. The `OperationsMetadata` is missing (I'm not sure if they are imported) and we get an error calling this function 
` options = optionsFromCapabilities(result, {
              layer: layerArray[0] || '',
              matrixSet: layer.attributes.tileMatrixSet
            })`

Reply:
This layer works for testing: https://isk.geobasis-bb.de/mapproxy/webatlasde_wmts/service?service=WMTS&request=GetCapabilities&REQUEST=GetCapabilities